app/models/post.rb

```diff
@@ -0,0 +1,20 @@
+class Post < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id', counter_cache: :posts_counter
```

Comment:
- [ ] I also could not create a post on the terminal due to incorrect association here. The post belongs to an `author`, as you have an `author_id` in your schema, not `user`.

Reply:
From my end this is the format to create a new post 

third_user = User.create(name: 'Hyda', photo_link: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Coder from Poland.', posts_counter: 0)

Reply:
Hi @hydaline-code This is a user not a post.

Reply:
ohhh 🤦‍♂️you are right  yeah sure ... i just lost focus while replying  okay this is a post creation  
first_post = Post.create(author_id: first_user.id, title: 'Hello2', text: 'This is my second post')